app/main.py

```diff
@@ -18,26 +21,31 @@ def __init__(
             self,
             start: tuple[int, int],
             end: tuple[int, int],
-            is_drowned: bool = False) -> None:
+            is_drowned: bool = False
+    ) -> None:
 
         self.is_drowned = is_drowned
         if start == end:
-            self.decks = [Deck(start[0], end[1])]
+            self.decks = [
+                Deck(start[0], end[1])
+            ]
         elif start[0] == end[0]:
-            self.decks = [Deck(start[0], i)
-                          for i in range(start[1], end[1] + 1)]
+            self.decks = [
+                Deck(start[0], i)
+                for i in range(start[1], end[1] + 1)
+            ]
         elif start[1] == end[1]:
-            self.decks = [Deck(i, start[1])
-                          for i in range(start[0], end[0] + 1)]
-
-    def get_deck(self, row: int, column: int) -> Deck:
+            self.decks = [
+                Deck(i, start[1])
+                for i in range(start[0], end[0] + 1)
+            ]
 
+    def get_deck(self, row: int, column: int) -> Optional[Deck]:
         for deck in self.decks:
             if deck.row == row and deck.column == column:
                 return deck
 
     def fire(self, row: int, column: int) -> None:
-
         deck = self.get_deck(row, column)
```

Comment:
Check if deck is None before, because it possible to raise an Error
If deck is none just exit from method

Reply:
done